Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -412,19 +428,24 @@ - (HRESULT)_asyncHttpOperation:(AsyncHttpOperation*)pOperation completedWithStat
 
                 // The location header can specify a relative or absolute URL: attempt to join it with the request URL.
                 NSURL* targetUrl = [NSURL URLWithString:Strings::WideToNSString(uriString.Get()) relativeToURL:[request URL]];
-                NSURLRequest* req = [NSURLRequest requestWithURL:targetUrl];
-                [self.client URLProtocol:self wasRedirectedToRequest:req redirectResponse:nsResponse];
+                __block StrongId<NSURLRequest> req = [NSURLRequest requestWithURL:targetUrl];
```

Comment:
It's safe to just capture things in blocks normally; Strong semantics are preserved since blocks referring to `id` types will retain/release.